app/grandchallenge/components/models.py

```diff
@@ -534,7 +539,7 @@ def default_field(self):
             return forms.JSONField
 
     @property
-    def file_mimetypes(self):
+    def file_mimetypes(self):  # noqa:C901
```

Comment:
Can be refactored to use a map, e.g. `INTERFACE_KIND_TO_MIMETYPE`, rather than lots of if statements.